### PR TITLE
Fix content type

### PR DESCRIPTION
**Pull request by btzr-io**
### Changes
- Display full `content-type`.
- Fix for https://github.com/lbryio/lbry-app/issues/863.

## Todo
- [ ] Fix build / conflicts
- [ ] Better fallback for `contentType`, see [mime-types](https://github.com/jshttp/mime-types)

### Related
@kauffj I tested some claims and all of them return the correct `content-type`, 
I'm still not sure if this also fixes https://github.com/lbryio/lbry/issues/1028 or is it a different issue?

### CODE REVIEW

File: src/renderer/lbry.js [hunk 1]
```diff
@@ -174,24 +174,21 @@ Lbry.publishDeprecated = (params, fileListedCallback, publishedCallback, errorCa
 
 Lbry.imagePath = file => `file://${__static}/img/${file}`;
 
-Lbry.getMediaType = (contentType, fileName) => {
-  if (contentType) {
-    return /^[^/]+/.exec(contentType)[0];
-  } else if (fileName) {
+// This isn't used anymore: remove it?
+Lbry.getMediaType = (fileName = '') => {
```
Comment (btzr-io) [inline src/renderer/lbry.js]: Should I remove this?

Comment (btzr-io) [reply src/renderer/lbry.js]: https://github.com/lbryio/lbry-app/pull/1172#discussion_r176751890

**Comment by tzarebczan** [comment]
@btzr-io any reason you closed this? Looks like https://github.com/lbryio/lbry-app/pull/1304 might address it as well.

**Comment by btzr-io** [comment]
@tzarebczan  not really, anyway I think is better to use an external module like [mime-types](https://github.com/jshttp/mime-types).

**Comment by btzr-io** [comment]
@tzarebczan https://github.com/lbryio/lbry-app/pull/1333

**Pull request closed without merging**